cipher_suite.go

```diff
@@ -115,10 +115,10 @@ func cipherSuiteForID(id CipherSuiteID) cipherSuite {
 // CipherSuites we support in order of preference
 func defaultCipherSuites() []cipherSuite {
 	return []cipherSuite{
-		&cipherSuiteTLSEcdheRsaWithAes256CbcSha{},
-		&cipherSuiteTLSEcdheEcdsaWithAes256CbcSha{},
 		&cipherSuiteTLSEcdheRsaWithAes128GcmSha256{},
 		&cipherSuiteTLSEcdheEcdsaWithAes128GcmSha256{},
```

Comment:
https://github.com/pion/dtls/issues/199#issuecomment-589863610
In Chromium, ECDSA is prior to RSA?

Reply:
:+1: will put ECDSA first.